packages/cmrjs/src/cmr-utils.js

```diff
@@ -44,6 +44,27 @@ function getS3UrlOfFile(file) {
   throw new Error(`Unable to determine location of file: ${JSON.stringify(file)}`);
 }
 
+function getFilename(file) {
```

Comment:
I hate this so much. It's not your fault, but having to check in five places to figure out a filename on what should be a uniform type? That's awful. Maybe worth an architecture meeting discussion.

Reply:
Yeah, as far as I know we can't predict at least if we'll have `fileName` or `name`. And I think the same with filename/filepath/key, because the schema was changed at some point but there could still be old records with or without those properties?